### PR TITLE
Update Travis CI integration

### PR DESCRIPTION
**Pull request by BanzaiMan**
Travis CI has announced Yarn support on 2016-11-21. https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn

This update describes the current support status of Travis CI on that date.

### CODE REVIEW

File: en/docs/_ci/travis.md [hunk 1]
```diff
@@ -14,5 +24,14 @@ cache:
 
 {% include_relative _ci/deb-specific-version.md %}
 
-> Let Travis CI know if you want Yarn to be
-> [installed by default](https://github.com/travis-ci/travis-ci/issues/6720).
+## container-based builds
+
+Container-based builds do not have the `sudo` privilege, so they must rely on other means to install.
```
Comment (Daniel15) [inline en/docs/_ci/travis.md]: I wish this wasn't necessary... Why do TravisCI containers not have `sudo` privilege?

**Comment by Daniel15** [comment]
Thanks!

File: en/docs/_ci/travis.md (hunk 1 above)
Comment (BanzaiMan) [inline en/docs/_ci/travis.md]: Because `sudo` inside the Docker container is a security risk right now.

Comment (Daniel15) [inline en/docs/_ci/travis.md]: @BanzaiMan - Why so? A Docker container should be reasonably contained such that even superuser permissions can't affect anything other than the container itself. CircleCI let people use `sudo` and also SSH into their containers 😄

**Pull request merged**